avaframe/com1DFA/DFAfunctionsCython.pyx

```diff
@@ -701,6 +706,14 @@ def updatePositionC(cfg, particles, dem, force, fields, int typeStop=0):
   cdef double peakKinEne = particles['peakKinEne']
   cdef double peakForceSPH = particles['peakForceSPH']
   cdef double totKForceSPH = particles['forceSPHIni']
+  cdef long long[:] ID = particles['ID']
+  # Stopped particles
+  cdef double[:] xStoppedArray = np.empty(0)
```

Comment:
Please initialise them like the arrays mNewArray below. -> avoids problems on Windows  (i.e. with the dtype given)

Reply:
Yes